charts/k8s-watcher/values.yaml

```diff
@@ -48,6 +48,10 @@ helm:
   enableActions: true
   volumeSizeLimit: "256Mi"
 
+network-mapper:
+  enabled: false
```

Comment:
Merging as is is fine while disabled as default.
For the next part, I'd consider before changing the default to true (if we desire that at any point) to make sure our CI tests the installation with it both enabled/disabled on existing AND new installations of the chart.